index.js

```diff
@@ -78,67 +70,28 @@ submitbtn.addEventListener('click', () => {
   title.value = '';
 });
 
-function removeme(id) {
-  Book.removeBook(Number(id));
-  books = Book.getBooks();
-  localStorage.setItem('books', JSON.stringify(books));
-  Mybooks();
-}
-
-const Links = document.querySelectorAll('.links');
-Links.forEach((link) => {
-  link.addEventListener('click', () => {
-    const sections = document.querySelectorAll('section');
-    sections.forEach((sec) => {
-      sec.classList.add('hide');
-      if (sec.classList.contains('showElement')) {
-        sec.classList.remove('showElement');
-      }
+const removeBook = (Book, mybooks) => {
+  const allBooks = document.querySelectorAll('.remove-btn');
+  allBooks.forEach((oneBook) => {
+    oneBook.addEventListener('click', (e) => {
+      Book.removeBook(Number(e.target.id));
+      const books = Book.getBooks();
+      localStorage.setItem('books', JSON.stringify(books));
+      mybooks(books);
+      window.location.reload();
     });
-    const section = document.querySelector(`section.${link.classList[1]}`);
-    section.classList.add('showElement');
   });
-});
-
-/*
-
-const aa = document.querySelector('.add-section');
-const ll = document.querySelector('.list-section');
-const cc = document.querySelector('.contact-section');
-const lissec = document.querySelector('.books', '.list-section')
-const lisadd = document.querySelector('.add', '.add-section')
-const liscon = document.querySelector('.contact', '.contact-section')
+};
 
-ll.addEventListener('click', () => {
-  lissec.classList.add('showElement');
-  lissec.classList.remove('hide');
-  lisadd.classList.add('hide');
-  lisadd.classList.remove('showElement');
-  liscon.classList.add('hide');
-  liscon.classList.remove('showElement');
-});
-
-aa.addEventListener('click', () => {
-  lissec.classList.add('hide');
-  lissec.classList.remove('showElement');
-  lisadd.classList.add('showElement');
-  lisadd.classList.remove('hide');
-  liscon.classList.add('hide');
-  liscon.classList.remove('showElement');
-});
-
-cc.addEventListener('click', () => {
-  lissec.classList.add('hide');
-  lissec.classList.remove('showElement');
-  lisadd.classList.add('hide');
-  lisadd.classList.remove('showElement');
-  liscon.classList.add('showElement');
-  liscon.classList.remove('hide');
-});
-*/
+const now = DateTime.now();
+const dateTime = document.querySelector('.current-date');
+dateTime.innerHTML = `Local date: ${now.toLocaleString(DateTime.DATE_FULL)} - Local time: ${now.hour}:${(now.minute)}:${(now.second)} hrs.`;
 
+/*
 const dateTime = document.querySelector('.current-date');
 dateTime.innerHTML = `${new Date().toLocaleDateString()},  ${new Date().toLocaleTimeString()}`;
+*/
```

Comment:
## Optional
- [ ] Please clean these commented code because it does not make your code clean

Reply:
Thank you for your review @OLIPLICHE 

Comments were already cleaned from code.

Regards,